Tests/MongoSwiftSyncTests/UnifiedTestRunner/SpecialTestOperations.swift

```diff
@@ -0,0 +1,40 @@
+import MongoSwiftSync
+
+struct UnifiedFailPoint: UnifiedOperationProtocol {
+    /// The configureFailpoint command to be executed.
+    let failPoint: BSONDocument
+
+    /// The client entity to use for setting the failpoint.
+    let client: String
+
+    static var knownArguments: Set<String> {
+        ["failPoint", "client"]
```

Comment:
I just listed out the values here because, in order to use `Mirror`, you actually need an instance of the type. I guess I could have created an instance with dummy values for both of the properties, but given there are a small number it seemed OK to just write them out.

same applies for rest of this file